### PR TITLE
[Maintenance] Refactor services' ids according to the new convention

### PR DESCRIPTION
**Pull request by Wojdylak**
| Q               | A
| --------------- | -----
| Branch?         | 1.7
| Bug fix?        | no
| New feature?    | no
| Related tickets | n/a

### CODE REVIEW

File: src/Resources/config/services/api.xml [hunk 1]
```diff
@@ -1,34 +1,53 @@
...
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd"
+>
     <parameters>
         <parameter key="sylius.pay_pal.request_trials_limit">5</parameter>
         <parameter key="sylius.paypal.logging_level_increased">true</parameter>
     </parameters>
     <services>
         <service id="Http\Discovery\Psr18Client" class="Http\Discovery\Psr18Client" />
+        <service id="sylius_paypal.http.discovery.psr_18_client" alias="Http\Discovery\Psr18Client" />
```
Comment (GSadee) [inline src/Resources/config/services/api.xml]: Couldn't we use `sylius.http_client` instead? 🤔

File: src/Console/Command/CompletePaidPaymentsCommand.php [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                $paymentDetails = $payment->getDetails();
+                $paymentDetails['status'] = StatusAction::STATUS_COMPLETED;
+
+                $payment->setDetails($paymentDetails);
+            }
+        }
+
+        $this->paymentManager->flush();
+
+        return 0;
```
Comment (GSadee) [inline src/Console/Command/CompletePaidPaymentsCommand.php]:
```suggestion
        return Command::SUCCESS;
```

File: src/Console/Command/CompletePaidPaymentsCommand.php [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+        }
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setName('sylius:pay-pal-plugin:complete-payments')
+            ->setDescription('Completes payments for completed PayPal orders')
+        ;
+    }
```
Comment (GSadee) [inline src/Console/Command/CompletePaidPaymentsCommand.php:63]: It could be moved to the attribute

File: src/Resources/config/services/api.xml [hunk 4]
```diff
@@ -1,34 +1,53 @@
...
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<container
+        xmlns="http://symfony.com/schema/dic/services"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd"
```
Comment (GSadee) [inline src/Resources/config/services/api.xml]: Wrong indentation

File: src/Console/Command/CompletePaidPaymentsCommand.php (hunk 3 above)
Comment (Wojdylak) [reply src/Console/Command/CompletePaidPaymentsCommand.php:63]: It will be done in 2.0 version.

**Review by GSadee — APPROVED** [review]

**Pull request merged**